portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,10 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function callServlets() {
```

Comment:
One last nit! callServlets() is still kind of general and its only purpose is to just fetch a random number, so what about naming this `fetchRandomNumber()`?

Reply:
no sooner said than done